manager/src/main/java/com/redhat/service/smartevents/manager/api/user/ProcessorsAPI.java

```diff
@@ -151,4 +170,32 @@ public Response deleteProcessor(@PathParam("bridgeId") String bridgeId, @PathPar
         processorService.deleteProcessor(bridgeId, processorId, identityResolver.resolve(jwt));
         return Response.accepted().build();
     }
+
+    /**
+     * Validate weather user is allowed to create new processor instance under an bridge
+     * 
+     * @param orgId Organisation id.
+     * @param bridgeId Bridge id.
+     */
+    private void validateServiceLimitToCreateNewProcessor(String orgId, String bridgeId) {
+        OrganisationServiceLimit organisationServiceLimit = limitService.getOrganisationServiceLimit(orgId);
+        long maxAllowedProcessors = organisationServiceLimit.getProcessorLimit();
+        long existingProcessorCount = processorService.getProcessorsCount(bridgeId);
```

Comment:
The `count` here is for ALL `Processor`'s including Error Handlers.

I doubt the Customer will want to pay for the `Processor`'s we use internally to support Error Handling?

@robpblake @danielezonca We provision a RHOSAK topic for the Error Handler. Is that billable (to us?!?).

The limit check here includes `SINK`, `SOURCE` and `ERROR_HANDLER` `Processor`'s.

Reply:
@manstis The current model is that each RHOSAK cluster is a cost of goods to us for providing the service. We will use a single RHOSAK cluster to host the topics for multiple customers to reduce our per-customer cost, and hence the cost of RHOSE to the customer. The end result of all that being that (with the current profile of the service) there is no visibility or cost to the end customer of the underlying RHOSAK cluster, it's "built in" to what they end up paying to use RHOSE.

You're right to call out that the check of Processor limit including the `ERROR_HANDLER` being incorrect. I'm still wondering if we actually need a specific `ERROR_HANDLER` type of Processor (to me it just looks like a Sink), but we have other discussions on that.

cheers,

Rob

Reply:
I added the filters to count processors of type SINK and SOURCE only.